lib/loader/loader.c

```diff
@@ -207,29 +206,29 @@ struct evmc_instance* evmc_load_and_create(const char* filename,
 
     enum evmc_loader_error_code ec = EVMC_LOADER_SUCCESS;
 
-    struct evmc_instance* instance = create_fn();
-    if (!instance)
+    struct evmc_vm* vm = create_fn();
+    if (!vm)
     {
-        ec = set_error(EVMC_LOADER_INSTANCE_CREATION_FAILURE,
-                       "creating EVMC instance of %s has failed", filename);
+        ec = set_error(EVMC_LOADER_VM_CREATION_FAILURE, "creating EVMC VM of %s has failed",
```

Comment:
I'm not sure about this error message.

Reply:
would be great if it could mention the name of the function it called, but it doesn't look easy